src/gui/screen_printing_serial.cpp

```diff
@@ -10,8 +10,38 @@
 #include "stm32f4xx_hal.h" //HAL_GetTick
 #include "screens.h"
 
+#define BUTTON_TUNE       0
+#define BUTTON_PAUSE      1
+#define BUTTON_DISCONNECT 2
+
 #pragma pack(push)
-#pragma pack(1)
+
```

Comment:
leaving this #pragma pack(push) alone here is one of the fastest ways to hell - please remove it. You are missing the #pragma pack(pop) for this one!